queries/julia/injections.scm

```diff
@@ -1,8 +1,13 @@
-; TODO: re-add when markdown is added.
-; ((triple_string) @markdown
-;   (#offset! @markdown 0 3 0 -3))
+((string_literal) @markdown
+ (#match? @markdown "^\"\"\"")
+ (#offset! @markdown 0 3 0 -3))
 
 [
   (line_comment)
   (block_comment)
 ] @comment
+
+((prefixed_string_literal
```

Comment:
Should regex really be injected into all raw-strings?

Reply:
`typeof(r"foo")` is `Regex` according to the [manual](https://docs.julialang.org/en/v1/manual/strings/#man-regex-literals).

Reply:
@theHamsta look further below: it only injects into strings prefixed with `r`

Reply:
Sorry, for the confusion. I thought it would be similar to Python r-strings